Map/Source/Field.cpp

```diff
@@ -0,0 +1,13 @@
+#include "Field.hpp"
+
+Field::Field() : printability(false)
+{
+}
+bool Field::isPrintable()
+{
+    return printability;
```

Comment:
ustalmy, ze pola klasy zaczynamy od _. Czyli tu będzie _printability. Można zmienic nazwę na _isPrintable, ale to juz zdecyduj sam

Reply:
i to w sumie jest zla nazwa. Bo pole zawsze będzie rysowane, ale nie zawsze będzie widoczne. Zmieniłbym to na _isVisible. Jak zmienisz nazwę pola, to musisz tez zmienic nazwy metod.